devsetup/Makefile

```diff
@@ -128,6 +128,7 @@ DATAPLANE_PLAYBOOK ?= osp.edpm.download_cache
 DATAPLANE_CUSTOM_SERVICE_RUNNER_IMG ?=quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest
 BM_NETWORK_NAME            ?=default
 BM_INSTANCE_NAME_PREFIX    ?=edpm-compute
+BM_INSTANCE_MEMORY         ?=8192
```

Comment:
On second thought, changing this default here would also change it for BMAAS (ironic) use-case, which we probably don't want. Maybe just changing it for `edpm_baremetal_compute` target would be better.